x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/data/SingletonOrdinalsBuilderTests.java

```diff
@@ -122,6 +122,37 @@ protected DriverContext breakingDriverContext() { // TODO move this to driverCon
         return new DriverContext(bigArrays, factory);
     }
 
+    public void testAllNull() throws IOException {
+        BlockFactory factory = breakingDriverContext().blockFactory();
```

Comment:
I am not sure I understand the purpose of this test?

Reply:
Ah, sorry. It's only vaguely related. I was investigating this issue and thought that the cause was ordinals not stored fields. But it turns out that ordinals were fine. Still, I wanted to make sure this was all true. Just that when you use the ordinals builder if you only give it nulls it'll build something that properly declares things as null.